Flutter/pdf-viewer/right-to-left.md

```diff
@@ -120,4 +120,6 @@ Then, import the `flutter_localizations` library, specify [localizationsDelegate
   }
 
 {% endhighlight %}
-{% endtabs %}
\ No newline at end of file
+{% endtabs %}
```

Comment:
Please API link for this too. General words may cause confusion 
<img width="484" alt="image" src="https://github.com/user-attachments/assets/4abd4571-a61a-45ca-b1fd-56453e133ec2">

Reply:
Added API to avoid confusion.